js/draw.js

```diff
@@ -1,78 +0,0 @@
-function draw() {
-  let darkGreen = color("rgb(17, 40, 21)");
-  background(darkGreen);
-
-  if (toggle == 1) {
```

Comment:
=== !!!

Reply:
what does it mean if toggle === 1? 
if reading this in a month or a year... you will not know anymore. Hence -> better naming. maybe extract the boolean and give it a value such as -> 
```js 
const gameJustStarted = toggle === 1
if (gameJustStarted) showScrollingText();
```

Reply:
also, if you are using toggle as a boolean value -> toggle should be a boolean, not 0 or 1. the nice thing about boolean values being that you can easily toggle them like so:
`toggle = !toggle`

Reply:
also, the more I think about it the less I like the name. toggle is too generic a name for a global variable. What are we toggling? then name it toggleForX or xToggle or something like that.

Reply:
or something like - gameStarted.

Reply:
also, the text keeps scrolling I think. wasting precious CPU resources on something that isn't shown, because it long moved out of screen.